src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+@import '/src/styles/blocks/variables';
```

Comment:
Ensure that the import path for the variables file is correct. The path should be relative to the current file's location. If the file is not found, it will cause a compilation error.